OpenRA.Mods.Common/Widgets/Logic/Installation/InstallFromDiscLogic.cs

```diff
@@ -54,7 +54,7 @@ enum Mode { Progress, Message, List }
 		Mode visible = Mode.Progress;
 
 		[ObjectCreator.UseCtor]
-		public InstallFromDiscLogic(Widget widget, ModContent content, Dictionary<string, ModContent.ModSource> sources, Action afterInstall)
```

Comment:
There's a commented out line with `afterInstall` you can remove. Additionally, it is passed via `WidgetArgs` in `ModContentLogic`. i.e. just search for the string `afterInstall` and nuke it.

Reply:
Removed it completely.